app/models/notification_phone_configuration.rb

```diff
@@ -0,0 +1,27 @@
+class NotificationPhoneConfiguration < ApplicationRecord
+  include EncryptableAttribute
+
+  belongs_to :in_person_enrollment, inverse_of: :notification_phone_configuration
+
+  encrypted_attribute(name: :phone)
+
+  def formatted_phone
+    Phonelib.parse(phone).international
+  end
+
+  def masked_phone
+    return '' if phone.blank?
+
+    formatted = Phonelib.parse(phone).national
+    formatted[0..-5].gsub(/\d/, '*') + formatted[-4..-1]
+  end
```

Comment:
I think this was copy-pasted from phone_configrations, what if we updated `PhoneFormatter` class to have a `masked_phone` method so we could share impls?

Reply:
That makes sense to me

Reply:
Added in 5f06653d50006e1c3cb82e0f7172502f7ff6b315